streams/lt.m3u

```diff
@@ -19,3 +19,17 @@ https://radijas.lrt.lt/radijas/master.m3u8
 https://radijas.lrt.lt/klasika/master.m3u8
 #EXTINF:-1 tvg-id="LRTOpus.lt",LRT Opus (1080p)
 https://radijas.lrt.lt/opus/master.m3u8
+#EXTINF:-1 tvg-id="LNK.lt",LNK (1080p)
+https://live.lnk.lt/lnk_live/smil:lnk_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259096&tokenhash=iOBCxbO8M6HzvpKYFaBxWNZcCdDiYfMk2sVLDu_Jx6MHIvOIOUESXwJRGYHau7Q3j70CiTayc3axjxix4WSktA==
+#EXTINF:-1 tvg-id="BTV.lt",BTV (1080p)
+https://live.lnk.lt/lnk_live/smil:btv_live.smil/playlist.m3u8?tokenstarttime=0&tokenendtime=1683259326&tokenhash=Z-qCE9n27ryLgi3p8uaIqwObuY5Eym1DHVoIpfmGKW6VjN-sjqsiboIh34wlyt3RVe49Vrx64giMMAj0TCJqeQ==
+#EXTINF:-1 tvg-id="LRTTV.lt",LRT HD (1080p)
+https://lrthd.lrt.lt/secure/lrt_hd/TK8DdmrYi1ahTMKCxVOdFg/1683237332/master.m3u8
+#EXTINF:-1 tvg-id="LRTPlius.lt",LRT Plius (1080p)
+https://plius.lrt.lt/secure/plius/0bxGmNnqOX1sQp9NbmVL6Q/1683237472/master.m3u8
```

Comment:
offline
<img width="612" alt="image" src="https://user-images.githubusercontent.com/7253922/236336551-d49684c1-c0a7-40a2-ac87-6cb85db9df29.png">

Reply:
>Works for me in VLC. Is it possible it's geo-blocked, but showing up as offline in the test? If so, I'll add the geo-blocked label.

same with this playlist

Reply:
Same here : https://www.lrt.lt/servisai/stream_url/live/get_live_url.php?channel=LTV2